lib/resources/example_json/dstu2_examples_relatedperson.rb

```diff
@@ -174,158 +176,76 @@ module Resources
                   "text": "Federated Person Principal"
                 },
                 "system": "urn:oid:2.16.840.1.113883.3.13.6",
-                "value": "URN:CERNER:IDENTITY-FEDERATION:REALM:AC193C1A-9763-45E1-9FA2-C1C3AA4BA16D:PRINCIPAL:PI98N2FK5TN",
+                "value": "urn:cerner:identity-federation:realm:687f29dd-69dd-4de5-acb1-fd8a2241ef3a:principal:EC4Ax54P8GI",
                 "period": {
-                  "start": "2016-01-01T10:00:00.000Z"
+                  "start": "2016-11-01T10:00:00.000Z"
                 }
               }
             ],
             "patient": {
-              "reference": "Patient/14067775",
-              "display": "NOAH, SALLY"
-            },
-            "relationship": {
-              "coding": [
-                {
-                  "system": "http://hl7.org/fhir/v3/RoleCode",
-                  "code": "DAUC",
-                  "display": "daughter",
-                  "userSelected": false
-                }
-              ],
-              "text": "Daughter"
+              "reference": "Patient/3838011",
+              "display": "Peters, Timothy"
```

Comment:
@joykoduru I just realized that I was interpreting this incorrectly when I asked you to change this in the PR.  I didn't realize that this was a RelatedPerson example, so the display here should remain as Jennifer whoever.